pkg/controller/volume/persistentvolume/controller.go

```diff
@@ -964,7 +964,7 @@ func (ctrl *PersistentVolumeController) recycleVolumeOperation(arg interface{})
 	volume = newVolume
 
 	// Find a plugin.
-	spec := vol.NewSpecFromPersistentVolume(volume, false)
+	spec := vol.NewSpecFromPersistentVolume(volume, false, "")
```

Comment:
yeah I don't remember why I didn't use the one from the PVC... I'm going to remove the `namespace` argument from NewSpecFromPersistentVolume and use pv.Namespace instead. This should reduce the patch size and avoid redundant information.